pages.ko/common/act.md

```diff
@@ -26,3 +26,7 @@
 - 자세한 로그 표시:
 
 `act -v`
+
+- 특정 워크플로우 실행:
+
+`act push -W {{path/to/workflow}}`
```

Comment:
Thanks for your contribution and welcome to tldr, Can you translate this placeholder too?

Reply:
@kbdharun Sure! Can you review it again?